portfolio/src/main/webapp/index.html

```diff
@@ -28,6 +28,9 @@
     <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
     <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
 
+    <!-- week2 JS fetching -->
+    <script type="text/javascript" src="script.js"></script>
+
```

Comment:
Nit:
Extra new line compared to other spacing in this file.

Reply:
fixed!